app/src/main/java/lt/vilnius/tvarkau/MainActivity.java

```diff
@@ -0,0 +1,165 @@
+package lt.vilnius.tvarkau;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v4.view.MenuItemCompat;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.SearchView;
+import android.support.v7.widget.Toolbar;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.Toast;
+
+import com.mikepenz.materialdrawer.AccountHeader;
+import com.mikepenz.materialdrawer.AccountHeaderBuilder;
+import com.mikepenz.materialdrawer.Drawer;
+import com.mikepenz.materialdrawer.DrawerBuilder;
+import com.mikepenz.materialdrawer.model.ProfileDrawerItem;
+import com.mikepenz.materialdrawer.model.interfaces.IDrawerItem;
+import com.mikepenz.materialdrawer.model.interfaces.IProfile;
+
+import butterknife.Bind;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+import lt.vilnius.tvarkau.fragments.ProblemsListFragment;
+import lt.vilnius.tvarkau.fragments.ProblemsMapFragment;
+
+/**
+ * An activity representing a list of Problems. This activity
+ * has different presentations for handset and tablet-size devices. On
+ * handsets, the activity presents a list of items, which when touched,
+ * lead to a {@link ProblemDetailActivity} representing
+ * item details. On tablets, the activity presents the list of items and
+ * item details side-by-side using two vertical panes.
+ */
+public class MainActivity extends AppCompatActivity implements Drawer.OnDrawerItemClickListener, AccountHeader.OnAccountHeaderListener, SearchView.OnQueryTextListener {
+
+    @Bind(R.id.toolbar)
+    Toolbar toolbar;
+
+    SearchView searchView;
+
+    Drawer drawer;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.main_activity);
+
+        ButterKnife.bind(this);
+
+        setSupportActionBar(toolbar);
+
+        setFrameFragment(R.id.main_nav_problems_list);
+        setNavigationDrawer(savedInstanceState);
+    }
+
+    public void setNavigationDrawer(Bundle savedInstanceState) {
+        AccountHeader headerResult = new AccountHeaderBuilder()
+                .withActivity(this)
+                .withHeaderBackground(R.drawable.header)
+                .addProfiles(
+                        new ProfileDrawerItem().withName("Mike Penz").withEmail("mikepenz@gmail.com").withIcon(getResources().getDrawable(R.drawable.profile_photo))
```

Comment:
I was planning to fix this after introducing Google login. I will replace it with Anonymous.